todo.py

```diff
@@ -59,7 +64,8 @@ def main():
 
         if choice == '1':
             task = input("Enter the task: ")
-            todo_list.add_task(task)
+            due_date = input("Enter the due date: ")
+            todo_list.add_task(task, due_date)
```

Comment:
This may cause issues with other merges as it seems others wanted this to be an optional feature,
maybe you could allow the user while inputting the name to enter a break character and enter the date if they want to but not require it. Although I also guess that if you enter no date that would do the same thing. Just an idea.

ex: Enter a task: "Task1 date:10/20/2024"       here date: is a separating the name and date but it is not required

Reply:
I added a prompt asking the user if they want to add a due date or not.

Reply:
resolved conflicts, looks good!